trusted_os/ctl.go

```diff
@@ -115,6 +116,14 @@ func (ctl *controlInterface) Start() {
 		return
 	}
 
+	if !irq {
+		Control.Init()
+		Control.DeviceMode()
+		Control.Reset()
+
+		Control.Start(device)
```

Comment:
Does `Start()` not return, or is it ok that we do this `Init`/`Mode`/`Reset`/`Start` immediately followed by further calls to `Init`/`DeviceMode`/`EnableInterrupts`?

Reply:
Start never returns.

Reply:
Might be worth a comment to that effect (there's precedent for that elsewhere :)?

Reply:
Should be better in https://github.com/transparency-dev/armored-witness-os/pull/4/commits/5e8f038c18e10c19283617f063c8bbfcfa935a8d